src/web/app/pages-instructor/instructor-audit-logs-page/instructor-audit-logs-page.component.html

```diff
@@ -1,8 +1,8 @@
 <h1>Student Activity Logs</h1>
-<p>This page allows you to find when your students have accessed or submitted a particular feedback session for a given course.</p>
+<p>This page allows you to find when your students have accessed or submitted a particular feedback session for a given course. The earliest date you can search for is <b>30 days</b> before today.</p>
```

Comment:
We should propagate this validation over to the API level as well, instead of relying on the UI to do the validation